Bio/PDB/Chain.py

```diff
@@ -77,8 +76,7 @@ def __contains__(self, id):
         return Entity.__contains__(self, id)
 
     def __delitem__(self, id):
-        """
-        Arguments:
+        """Arguments:
```

Comment:
That's not a suitable docstring. How about:

"""Delete item.

Arguments:
...
"""